examples/routing/mini_gamma_IRF.jl

```diff
@@ -0,0 +1,58 @@
+using ClimaRivers
+using JSON, Dates
+
+# build hillslope model
+shape = 1.5
+timescale = 1.0
+t_max_hs = 60.0
+hillslope = MizurouteHillslopeV1{Float64}(shape, timescale, t_max_hs)
+
+# build channel model
+wave_velocity = 1.5 * 86400
+diffusivity = 8000 * 86400
+t_max_ch = 120.0
+channel = MizurouteChannelV1{Float64}(wave_velocity, diffusivity, t_max_ch)
+
+# build model
+river_model = HillslopeChannelRiverModel(hillslope, channel)
+
+# build static environment
+data_file_path = joinpath(@__DIR__,"..","..","mini_data","routing")
```

Comment:
Please use this type of local `joinpath` linking in the `gamma_IRF.jl` and `compare_results.jl` too!

Reply:
-  fixed in gamma_IRF, going to fix in compare_results